packages/playground-examples/copy/ko/4-0/New JS Features/Logical Operators and Assignment.ts

```diff
@@ -0,0 +1,41 @@
+// 2020년 버전의 JavaScript에서 논리 할당 연산자는 새로운 기능입니다.
+// 이 연산자는 JavaScript 객체를 편집하는
```

Comment:
```suggestion
// 이 연산자는 JavaScript 객체를 변경하는
```
편집 보다는 수정, 변경 이라는 표현은 어떨까요?

Reply:
edit를 그대로 직역했는데 '변경'이 더 자연스럽겠네요 :)
반영하겠습니다.